### DISCUSSION TITLE
google social login not working

### DISCUSSION THREAD

**Discussion by javaspeak** [reown-com/appkit#2501] [Q&A] [answered]
<img width="1063" alt="Screenshot 2024-07-03 at 10 56 12" src="https://github.com/WalletConnect/web3modal/assets/26520180/030859f4-2405-4ccd-9df5-75c68300d4fa">

So the pop up window on the left seems good, but the right says error.

Is this a known issue?

**Comment by javaspeak** [comment]
You can see the code here:

https://github.com/spotadev/wallet-connect-email-socials-login/blob/main/README.md

It is a small example just showing the problem. 

Also raised this issue:

https://github.com/WalletConnect/web3modal/issues/2502

I guess it is a bug but have also started this discussion in case it is an issue with configuration.

**Comment by javaspeak** [comment]
The 2 files which have the code in are:

https://github.com/spotadev/wallet-connect-email-socials-login/blob/main/src/main.tsx

https://github.com/spotadev/wallet-connect-email-socials-login/blob/main/src/App.tsx

**Comment by rtomas** [comment, marked as answer]
hi @javaspeak,

Thanks for reporting this issue. Please update your package @web3modal/ethers or @web3modal/wagmi to v5.0.6 to solve it.
